planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -518,11 +553,13 @@ void BehaviorPathPlannerNode::run()
 
   publishDebugMarker(bt_manager_->getDebugMarkers());
 
-  if (planner_data_->parameters.visualize_drivable_area_for_shared_linestrings_lanelet) {
+  if (planner_data->parameters.visualize_drivable_area_for_shared_linestrings_lanelet) {
     const auto drivable_area_lines = marker_utils::createFurthestLineStringMarkerArray(
-      util::getDrivableAreaForAllSharedLinestringLanelets(planner_data_));
+      util::getDrivableAreaForAllSharedLinestringLanelets(planner_data));
     debug_drivable_area_lanelets_publisher_->publish(drivable_area_lines);
   }
+
+  mutex_.unlock();
```

Comment:
This has to be removed.

Reply:
[15caaee](https://github.com/autowarefoundation/autoware.universe/pull/657/commits/15caaeec35c22b5493d7ccbcca6a74211fc17171)